script/setup/critools-version

```diff
@@ -1 +1 @@
-53ad8bb7f97e1b1d1c0c0634e43a3c2b8b07b718
+f9710f792992ae7264d83cc894621174454d719c
```

Comment:
can use `v1.24.0` now that it's been tagged

Reply:
ok, will update to use the tag (both here and in the 1.6 release one)